### DISCUSSION TITLE
Music Assistant Addon armv7 support

### DISCUSSION THREAD

**Discussion by giacomocarrozzo** [music-assistant/support#1132] [Q&A] [answered]
I would like to run the addon on my Raspberry PI 3 with armv7 architecture. Will it never get supported?
Thank you for your work

**Comment by OzGav** [comment]
The add on should run on the Pi3 as far as I know. Why do you think it doesn’t?

**Comment by giacomocarrozzo** [comment]
Because the architecture list doesn't have armv7:
https://github.com/music-assistant/home-assistant-addon/blob/main/music_assistant_beta/config.yaml#L3

**Comment by OzGav** [comment]
OK yeah I actually just had another query about that. I assume you are using the compute module? If so you are likely not going to meet the minimum requirements for MA V2. HA alone needs 1GB min and better with 2GB RAM. Then for MA:
Minimal rpi4 (or comparable), no rpi 3
64 bits cpu and OS
absolute minimum of 2gb memory, 4 gb or higher recommended

**Reply by giacomocarrozzo** [reply]
I'm using a raspberry pi 3b+. So I need to update my hardware :) Thank you for the quick reply

**Comment by marcelveldt** [comment, marked as answer]
A raspberry pi3 is way to lightweight for MA, especially if you are also running HA on it.

Minimum raspberry pi 4 with 2gb memory, more is adviced.
